post-processor/shell-local/post-processor.go

```diff
@@ -65,7 +67,7 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 	}
 
 	if p.config.ExecuteCommand == "" {
-		p.config.ExecuteCommand = "chmod +x {{.Path}}; {{.Vars}} {{.Path}}"
+		p.config.ExecuteCommand = "chmod +x {{.Script}}; {{.Vars}} {{.Script}} {{.Artifact}}"
```

Comment:
Can you make a note in the docs to reflect this fix? Also please note the issue you ran into so if someone wants to customize this command they do not make the same mistake we did.